### DISCUSSION TITLE
Swiper prev/next icons not showing in Safari and Chrome

### DISCUSSION THREAD

**Discussion by pandabrand** [nolimits4web/swiper#4991] [Q&A] [answered]
Hey all,
I have a weird situation that I haven't been able to get tot he bottom of. My swiper instances aren't showing the prev/next icons on Safari and Chrome (OS X) specifically. I haven't tried Chrome on Windows. Firefox works. I get no errors and its consistent across at least two versions (6.6.1 and 7.0.6). 
You can see an example here:  
https://pleinairarchive.com/painting-date/july-4-2021-annie-coleman-melissa-english-gizelle-jean-english/. 
The 'Artist Images' and 'Leslie Images' should have icons, if you hover in the right place you'll find the div for them and get a cursor.

**Comment by nolimits4web** [comment, marked as answer]
It is because of this CSS prop on `body` in your stylesheet:
```
text-rendering: optimizeSpeed;
```

**Reply by pandabrand** [reply]
Ahhh, this is in my reset file and I'm using a theme so I didn't even think to look there. Also, I've never used the `text-rendering` property. Thank you!
